Integrations/integration-GoogleVault.yml

```
@@ -0,0 +1,1551 @@
+            'Export Name': export_name,
+            'Created Time': create_time
+        }
+        markdown = tableToMarkdown(title, output_for_markdown, ['Matter ID', 'Export ID', 'Export Name', 'Created Time'])
+        demisto.results( {
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['text'],
+            'Contents': context_output,
+            'HumanReadable': markdown,
+            'EntryContext': {
+                'GoogleVault.Export(val.ID == obj.ID)': context_output
+            }
+        })
+
+
+    def get_export_command():
+        service = connect()
+        matter_id = demisto.getArg('matterId')
+        export_id = demisto.getArg('exportID')
+        try:
+            response = get_export_by_id(service, matter_id, export_id)
+            usr_msg = "Here are your export details:\nMatter ID: {}\nExport ID: {}\nExport Name: {}\nExporting Status: {}.".format(matter_id, export_id, response.get('name'), response.get('status'))
+            export_name = response.get('name')
+            export_status = response.get('status')
+            context_output = []
+            context_output.append({
+                'MatterID': matter_id,
+                'ID': export_id,
+                'Name': export_name,
+                'Status': export_status
+            })
+            demisto.results( {
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': context_output,
+            'HumanReadable': usr_msg,
+            'EntryContext': {
+                'GoogleVault.Export(val.ID == obj.ID)': context_output
+                             }
+            })
+        except Exception, ex:
+            demisto.results( { "Type" : entryTypes["error"], "ContentsFormat" : formats["text"], "Contents" : 'Unable to get export. Error: {}'.format(ex)})
+            sys.exit(2)
+
+
+    # @@@@@@@@ DEMISTO COMMANDS @@@@@@@@
+
+    if demisto.command() == 'test-module':
```

surround the next block with try-except to catch all exceptions instead of doing it inside each of the functions